b12x/attention/_shared/mla/decode_math.py

```
@@ -623,6 +802,12 @@ def s1_qk_nope_block_scaled(
             latent_scale_per_token=latent_scale_per_token,
             kv_sc_base_addr=kv_sc_base_addr,
         )
+    if cutlass.const_expr(scale_format in (3, 5)):
+        return s1_qk_nope_kvarn_bf16(
```

Ruff (0.16.1)

[error] Undefined name `s1_qk_nope_kvarn_bf16`

(F821)